frontend/src/metabase/query_builder/components/expressions/expressions.types.ts

```diff
@@ -0,0 +1,5 @@
+import { Expression, Field } from "metabase-types/types/Query";
+
+type ExpressionConfig = Expression | Field | Array<any>; // TODO: add proper types for all kinds of expressions
```

Comment:
There is an existing type for expressions https://github.com/metabase/metabase/blob/6265ac4fffe4889efa8175cb75fc78d10affaa1a/frontend/src/metabase-types/types/Query.ts#L302